fc/jira/triage_task.py

```diff
@@ -113,10 +114,12 @@ def create(self):
     def type_str(self) -> str:
         return 'Triage'
 
-    def score(self) -> int:
+    def score(self) -> tuple:
         score = self._calculate_score()
-        self._update_triage_vfr(score)
-        return score
+        self._update_triage_vfr(score)\
+            if self.score_value is None or self.score_value != score\
+            else cli_library.echo('not updating score for {} as new score is same'.format(self.id))
```

Comment:
I'm not sure if like having a `cli_library` call from inside a Jira class.  Plus you already have some output happening in `tasks. _score_triage_and_el_task`.